src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -157,7 +157,18 @@ internal sealed class MeterProviderSdk : MeterProvider
                         var metricStreamConfigs = new List<MetricStreamConfiguration>(viewConfigCount);
                         foreach (var viewConfig in this.viewConfigs)
                         {
-                            var metricStreamConfig = viewConfig(instrument);
+                            MetricStreamConfiguration metricStreamConfig = null;
+
+                            try
+                            {
+                                metricStreamConfig = viewConfig(instrument);
+                            }
+                            catch (ArgumentException ex)
+                            {
+                                OpenTelemetrySdkEventSource.Log.MetricInstrumentIgnored(instrument.Name, instrument.Meter.Name, "Applying View Configuration failed with error:" + ex.Message, "Fix the view configuration.");
```

Comment:
yes this allocates string, even if none listening. Could have added a different Event to avoid this, but I think this is acceptable, given that this is not hot path.

Reply:
Consider adding a TODO comment